resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -675,6 +676,20 @@ private[spark] object Config extends Logging {
       .checkValue(value => value > 0, "Maximum number of pending pods should be a positive integer")
       .createWithDefault(Int.MaxValue)
 
+  val KUBERNETES_JOB_MIN_MEMORY = ConfigBuilder("spark.kubernetes.job.min.memory")
+    .doc("The minimum memory for running the job, in MiB unless otherwise specified. This only " +
+      "applicable when you enable `VolcanoFeatureStep` feature step in" +
```

Comment:
To enable the step one has to provide the fully qualified class name. Would it be a good idea to use String interpolation and parameter `classOf[VolcanoFeatureStep].getName`

Reply:
> To enable the step one has to provide the fully qualified class name. Would it be a good idea to use String interpolation and parameter classOf[VolcanoFeatureStep].getName

Yep, this is a good idea.

> Since both KubernetesDriverConf and KubernetesExecutorConf may configure their own instances of VolcanoFeatureStep then would this create two PodGroups ?

No, only created before driver pod started. see note in [here](https://github.com/apache/spark/blob/f62b36c6d3964c40336959b129b284edb8097f61/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/KubernetesFeatureConfigStep.scala#L73), and pod group annotation is set in both driver and executor side.